src/tests/meetings.test.js

```diff
@@ -0,0 +1,194 @@
+const request = require('supertest');
+const server = require('../server');
+const Meetings = require('../database/helpers/meetings');
+
+const MEETING_API_URL = '/api/meetings';
+
+afterEach(async () => {
+    await Meetings.truncate();
+})
+
+beforeEach(async () => {
+    await Meetings.truncate();
+})
+
+describe('MEETINGS ROUTER', () => {
+    describe('GET ROUTE /MEETINGS', () => {
+        it('should return status 200 on success', async () => {
+            const res = await request(server).get(MEETING_API_URL);
+            expect(res.status).toEqual(200);
```

Comment:
make sure you actually create entries before you try to fetch anything because the table is going to be truncated before and after each test. Plus, remember to make a second test to check res.body as well.